### DISCUSSION TITLE
post/ifkakao2024-delayed-transfer/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#109] [Announcements]
# post/ifkakao2024-delayed-transfer/

Kafka 기반 지연이체 서비스를 재설계하고 개발한 경험과 if(kakao) 발표 소감을 공유합니다.

https://tech.kakaopay.com/post/ifkakao2024-delayed-transfer/

**Comment by LichKing-lee** [comment]
와~ 발표도 잘 들었는데 글로도 올라왔네요 좋은 공유 감사합니다!

**Comment by minsoozz** [comment]
좋은 글 감사합니다~

**Comment by kyungjunleeme** [comment]
좋은 글 감사합니다!
